terraform/20-app/locals.tf

```diff
@@ -27,6 +27,19 @@ locals {
   use_auto_scaling  = local.use_prod_sizing
   use_ip_allow_list = local.environment != "prod"
 
+  scheduled_autoscaling_policies_for_dev_environments = {
+    start_of_working_day_scale_out = {
+      min_capacity = 1
+      max_capacity = 1
+      schedule     = "cron(0 07 ? * MON-FRI *)"   # Run every weekday at 7am
+    }
+    end_of_working_day_scale_in = {
+      min_capacity = 0
+      max_capacity = 0
+      schedule     = "cron(0 20 ? * MON-FRI *)"   # Run every weekday at 8pm
+    }
+  }
+
```

Comment:
does this override what we have defined on the services?

``` 
   desired_count            = local.use_auto_scaling ? 3 : 1
   autoscaling_min_capacity = local.use_auto_scaling ? 3 : 1
   autoscaling_max_capacity = local.use_auto_scaling ? 20 : 1
```

Reply:
Yes it will do. When I wrote these policies it was on the premise that we'd only apply that to dev environments. But I can replicate this in the policy so that we're covered whether the policy is being applied to a dev or a test env